frontend/components/commons/forms/questions-form/QuestionsForm.component.vue

```diff
@@ -251,121 +242,110 @@ export default {
         default:
       }
     },
-    onChange({ newOptions, idComponent }) {
+    onChangeTextArea({ newOptions, idComponent }) {
+      this.inputs = this.inputs.map((input) => {
+        if (input.id === idComponent) {
+          input.options = [{ ...newOptions, value: newOptions.text }];
+        }
+        return input;
+      });
+    },
+    onChangeMonoSelection({ newOptions, idComponent }) {
       this.inputs = this.inputs.map((input) => {
         if (input.id === idComponent) {
           input.options = newOptions;
         }
         return input;
       });
     },
+    async sendBackendRequest(responseValues) {
+      try {
+        let responseData = null;
+        if (this.responseId) {
+          responseData = await this.updateRecordResponses(responseValues);
+        } else {
+          responseData = await this.createRecordResponses(
+            this.recordId,
+            responseValues
+          );
+        }
+        const { data: updatedResponses } = responseData;
+
+        if (updatedResponses) {
+          this.updateResponsesInOrm({
+            record_id: this.recordId,
+            ...updatedResponses,
+          });
+        }
+      } catch (error) {
+        console.log(error);
+
+        // NOTE - if there is a responseid for the input, means that it's an update. Otherwise it's a create
+        let message = "";
+
+        if (this.responseId) {
+          message = `There was a probleme to update response with responseId: ${this.responseId}`;
+        } else {
+          message = `There was a probleme to create response with recordId. ${this.recordId}`;
+        }
```

Comment:
Regarding your [old codebase](https://github.com/argilla-io/argilla/pull/2937/files#diff-618fb50ffd3e13cac0d657be02a45d03fcfc9b94e817626a4a5501694aab784fL469), the error message was the same. I would keep as it.
```suggestion
        const message = "There was a problem to save the response";
```